client/src/components/QnAcomponents/mainQnA.jsx

```diff
@@ -16,24 +16,11 @@ class QnA extends React.Component {
     };
     this.showMoreQuestions = this.showMoreQuestions.bind(this);
     this.search = this.search.bind(this);
-    this.getQuestionsList = this.getQuestionsList.bind(this);
   }
 
   componentDidMount() {
-    this.getQuestionsList()
-    var questionsToShow = sampleData.questions.results;
-    if (questionsToShow.length > 2) {
-      this.setState({
-        isMoreQuestionsButtonShown: true
-      });
-    }
-    questionsToShow = questionsToShow.slice(0, 2);
-    this.setState({
-      questions: questionsToShow
-    });
-    //get product name by its id
     let productId = this.props.productId;
-
+    //GET PRODUCT NAME BY ITS ID
```

Comment:
We might want to do this step in app.jsx or higher up in the hierarchy later down the line for optimization purposes. I think it'll be good for now though.